models/articles-model.js

```diff
@@ -1,16 +1,17 @@
 const db = require("../db/connection");
 
-exports.selectArticles = (article_id) => {
-  if (!Number(article_id)) {
+exports.selectArticles = (articleId) => {
+  if (!Number(articleId)) {
```

Comment:
You have psql error handlers that should deal with this.

Reply:
<img width="640" alt="Screenshot 2024-08-26 at 14 16 09" src="https://github.com/user-attachments/assets/166e8a8d-a3f8-4d23-b699-7431073e3dc0">